src/e2e/java/uk/gov/hmcts/reform/sscs/functional/ReissueAppellantAppointeeFurtherEvidenceHandlerFunctionalTest.java

```diff
@@ -29,9 +29,9 @@ private void verifyEvidenceIssued() {
         List<SscsDocument> docs = caseData.getSscsDocument();
         log.info("verifyEvidenceIssued ccdCaseId " + ccdCaseId);
 
-        assertNull(docs.get(0).getValue().getEvidenceIssued());
-        assertEquals("Yes",docs.get(1).getValue().getEvidenceIssued());
-        assertEquals("Yes",docs.get(2).getValue().getEvidenceIssued());
-        assertEquals("Yes",docs.get(3).getValue().getEvidenceIssued());
+        Assertions.assertThat(docs)
+            .extracting(SscsDocument::getValue)
+            .extracting(SscsDocumentDetails::getEvidenceIssued)
+            .contains("Yes", "Yes", "Yes");
```

Comment:
.contains("Yes", "Yes", "Yes")
can be simplied/made more elegant by doing:
.hasSize(3)
.containsOnly("Yes)

Reply:
your suggestion will limit the list of size is 3, it slightly change the original logic. The change is aim to check without the ordering but it is not about the size of the list. 

The code just have 3 Yes value. I think it is acceptable.

Reply:
I'll make a call here and say that while Lucas' suggestion at first glance appears to be marginally better in this particular context, it's still subjective and I'm happy to keep the current implementation.

I'll mark this conversation as resolved.